model/patch/patch.go

```diff
@@ -628,6 +640,25 @@ func (p *Patch) IsChild() bool {
 	return p.Triggers.ParentPatch != ""
 }
 
+func (p *Patch) SetParametersFromParent() error {
```

Comment:
I think we should add unit tests of this function

Reply:
added